### PR TITLE
Linked list insertions

### PR DESCRIPTION
**Pull request by bashamtg**

### CODE REVIEW

**Comment by bashamtg** [comment]
- [X] Top-level README “Table of Contents” is updated
 - [X] README for this challenge is complete
   - [x] Summary, Description, Approach & Efficiency, Solution
   - [ ] Picture of whiteboard
   - [X] Link to code
 - [X] Feature tasks for this challenge are completed
 - [X] Unit tests written and passing
   - [X] “Happy Path” - Expected outcome
   - [X] Expected failure
   - [X] Edge Case (if applicable/obvious)

File: python/data_structures/linked_list.py [hunk 1]
```diff
@@ -1,62 +1,107 @@
...
+        node = self.head
+
+        while node:
+            if node.value == target_value:
+                new_node.next = node.next
+                node.next = new_node
+                break
+            raise TargetError(f'{target_value} is missing')
+        if node is None:
+            raise TargetError('There are no nodes')
```
Comment (jstreifel-33) [inline python/data_structures/linked_list.py:65]: Your insert_after method does not actually traverse the linked list, and raises an exception if the target value is not the head value.

Revise the insert_after method to traverse the list, and function properly if inserting after a value other than head, such as the second or third value of the linked list.

I recommend creating a new test or two to ensure proper behavior in this case. Seems like a gap in the provided testing suite has been found.
